app/javascript/src/components/Dashboard/Contacts/Table/column.js

```diff
@@ -0,0 +1,3 @@
+import { buildContactsTableColumnData } from "./utils";
+
+export const CONTACTS_TABLE_COLUMN = buildContactsTableColumnData();
```

Comment:
https://www.loom.com/share/01c7bf8017a9473b9539604b0c3f6ba0

Reply:
Ok Yedhin, I have directly imported 
`import { buildContactsTableColumnData } from "./utils";`  and done required changes in **index.js** file as:
`columnData={buildContactsTableColumnData()}`
 And deleted **column.js** file.
Thanks.